angular-material/angular-material-tests.ts

```diff
@@ -4,7 +4,8 @@ var myApp = angular.module('testModule', ['ngMaterial']);
 
 myApp.config((
     $mdThemingProvider: ng.material.IThemingProvider,
-    $mdIconProvider: ng.material.IIconProvider) => {
+    $mdIconProvider: ng.material.IIconProvider,
+	$mdProgressCircularProvider: ng.material.IProgressCircularProvider) => {
```

Comment:
The pull request is still targeted to `types-2.0`. To target `master` you have to click "Edit" in the upper-right and change the base branch to `master`.
But this pull request no longer seems to have any commits by you! Hope you can recover your commits!